### DISCUSSION TITLE
Remove MagickScript from the API.

### DISCUSSION THREAD

**Discussion by dlemstra** [dlemstra/Magick.NET#604] [General]
I am considering to remove the MagickScript API from the library. I don't think that a lot of people use this API and I would like to remove it so I no longer need to maintain it. In the next release this will be marked as obsolete. And if I don't get a lot of responses to this it will be removed in a later release. This would also mean that there will be no new releases `Magick.NET.Web` library because this library was created on top of that API and only works in and old aspnet web project. Please respond here if you would want me to keep this API and explain how you are using it.

**Comment by rlalance** [comment]
Glad I could find this post.

I am about to restart a Proof of concept that will use MagickScript.

I am create a node based editor that will generate MagickScript that will be sent to a .net core server backend to run the script.

I also found the commit you started remove MagickScript, so I can fork for that commit if you end up removing it.

Thanks!

**Comment by dlemstra** [comment]
It has already been removed in June (https://github.com/dlemstra/Magick.NET/releases/tag/7.18.0.0). Instead of forking the project and still using MagickScript I would advise you to create your own version of MagickScript that talks with the current API. The code for MagickScript was generated with the following tool: https://github.com/dlemstra/Magick.NET/tree/0f7b45d212c1821c0764cd745522dbbfe94f119d/tools/FileGenerators/MagickScript. This tool is not compatible with the current API and that is why I stopped supporting it. I would advise you to copy that tool to your project and make it work again.

**Comment by rlalance** [comment]
Start from scratch is quite an undertaking. But, appreciate the leads. Thank you.
